pkg/vpcmodel/subnetsConnectivity.go

```diff
@@ -252,14 +255,45 @@ func (v *VPCsubnetConnectivity) computeAllowedConnsCombined() error {
 			v.AllowedConnsCombined[src][dst] = combinedConns
 		}
 	}
+	return nil
+}
 
+func (v *VPCsubnetConnectivity) computeStatefulConnections() error {
+	for src, endpointConns := range v.AllowedConnsCombined {
+		for dst, conns := range endpointConns {
+			if conns.IsEmpty() {
+				continue
+			}
+			dstObj := v.cloudConfig.NameToResource[dst.Name()]
+			var otherDirectionConn *common.ConnectionSet = nil
```

Comment:
initialized as `nil` by default in `var` declaration?

Reply:
right. fixed